characters.py

```diff
@@ -1,154 +1,158 @@
 import random
 from abc import ABC, abstractmethod
 import names
-import math
 
 import modifiers
 
 
```

Comment:
Trochę się rzeczywiście bałaganu narobiło w tych initach...
Muszę pomyśleć czy to rzeczywiście było słuszny ruch z perspektywy projektu...

Reply:
ale po zastosowaniu tych zmiennych klasowych się troszkę rozjasnilo ;p

Reply:
Ok. Przy założeniu, że wszyscy bohaterowie tej samej klasy mają na początku mieć identyczne parametry - możemy to wszystko wsadzić do zmiennych klasowych.
Tylko oczywiście WSZYSTKO ma być opisane w docstringach :)

Reply:
postaram sie troszkę jeszcze poopisywać